menuinst/platforms/win.py

```diff
@@ -64,6 +65,22 @@ def quick_launch_location(self) -> Path:
     def desktop_location(self) -> Path:
         return Path(windows_folder_path(self.mode, False, "desktop"))
 
+    @property
+    def terminal_profile_location(self) -> Path:
+        if self.mode == "system":
+            warnings.warn("Terminal profiles are not available for system level installs")
```

Comment:
This is a warning for the end user, right? In that case:

```suggestion
            log.warn("Terminal profiles are not available for system level installs")
```

I think I messed up the other `warnings.warn` calls to 🤔 They should have been `logging.Logger.warning` calls too.